js/private/js_binary.sh.tpl

```diff
@@ -183,7 +183,7 @@ if [ ! -f "$JS_BINARY__NODE_BINARY" ]; then
     logf_fatal "node binary '%s' not found in runfiles" "$JS_BINARY__NODE_BINARY"
     exit 1
 fi
-if [ ! -x "$JS_BINARY__NODE_BINARY" ]; then
+if [ "$_IS_WINDOWS" -ne "1" ] && [ ! -x "$JS_BINARY__NODE_BINARY" ]; then
```

Comment:
do you mean to leave these changes in??

Reply:
yes, apparently `-x` doesn't work for checking the "binary" status on Windows